src/js/apps/admin/program/workflows/workflows_app.js

```diff
@@ -20,6 +23,27 @@ export default App.extend({
     this.actions = actions;
 
     this.showChildView('content', new ListView({ collection: actions }));
+
+    if (_DEVELOP_) {
+      const lists = [];
+
+      lists.unshift({
```

Comment:
I don't think we'll need to `unshift` here I think we'll just be able to define the list with new action and new flow, and we can add the new flow option in this PR.    Really there's only one list here so we don't need to use lists at all.  This could be a collection like the publish state droplist which specifies test and icon.  You could also probably handle the click event like the `MainNavDroplist` where the event is handled on the component definition and calls `model.onSelect()`